README.md

```diff
@@ -0,0 +1,51 @@
+Install postresql server
+```
+apt install postgresql
+```
+
+Create a postgres user and db as root:
+```
+createuser <your username>
+createdb -O <your username> tunnistamo
+```
+
+Clone the repo
+
+```
+git clone git@github.com:City-of-Helsinki/tunnistamo.git
+cd tunnistamo
+```
+
+Initiate a virtualenv
+```
+pyenv virtualenv tunnistamo-env
+pyenv local tunnistamo-env
+```
+
+Install the Python requirements
+```
+pip install prequ
+rm requirements.txt
+prequ update
```

Comment:
The above three lines should only be done by developers actually wanting to update the requirements. Running `prequ` is not part of a normal development installation.